02.general/05.adguard-filter-policy/docs.en.md

```diff
@@ -60,26 +56,24 @@ If a rule is subject to the list of limitations described below, then it won’t
 
 If a rule is subject to the list of limitations described below, then it won’t be added to this filter.
 
-- Widgets that are a part of website’s functionality are not blocked.  
- Such as “Comments”, “Embedded Post”, “Surveys” widgets or authorization via social networks.
-- Links to website’s communities in social networks are not blocked.
+- Rules that block widgets that are a part of website’s functionality. Such as “Comments”, “Embedded Post”, “Surveys” widgets or authorization via social networks.
+- Rules that block links to website’s communities in social networks are not blocked.
 - Rules that cause problems with functioning of websites will be removed.
-- Rules that are specific to the particular website will be added only if the website has sufficient traffic.  
- Traffic is determined by open statistics (if it has such) or in other ways.  
- Website’s traffic is considered sufficient when it has 30 thousands visitors a month.
+- Rules that are specific to the particular website will be added only if the website has sufficient traffic. Traffic is determined by open statistics (if it has such) or in other ways. Website’s traffic is considered sufficient when it has 30 thousands visitors a month.
 
 
 ## Annoyances filter
 
 ##### What shall be blocked with this filter?
 
-- This filter shall block elements on the page that are not ads but obstruct view and make it harder to see the actual content of the website. Such elements as cookie notices, third-party widgets and in-page popups.
+- This filter shall block elements on the page that are not ads but obstruct view and make it harder to see the actual content of the website. Such elements as cookie notices, third-party widgets and in-page popups, email subscription forms and aggressively placed social media widgets.
 
 #### Limitations and Exceptions
 
 If a rule is subject to the list of limitations described below, then it won’t be added to this filter.
 
-- Any elements that are essential for the websites' functioning (e.g. authorization forms) will not be blocked even if they satisfy other requirements.
+- Rules that block elements that are essential for the websites' functioning (e.g. authorization forms) will not be blocked even if they satisfy other requirements.
+- Rules blocking elements in headers/footers.
```

Comment:
Actually we are blocking subscription forms and mobile app links in footer. It was about social links